### ISSUE TITLE
Check if Default image is loaded or not

### ISSUE THREAD

**Issue by elkhalifte** [tjoskar/ng-lazyload-image#522]
how can i check if the default image is loaded or failed.

from onStateChange i can only know if the lazyload image is loaded or not. 
i want to do the same for the default image

**Comment by tjoskar** [comment]
Hi @elkhalifte,

Right now, you can't, unfortunately. The idea on my part was that `default` image should only be used with an image you know exist and know the dimension of. 

Have you solved this in another way? Or is this still an issue for you? I believe it would be possible to implement this feature if you still have the need for it.

**Comment by elkhalifte** [comment]
Hi @tjoskar ,
 Yeah i really need this feature . Thank you for this amazing package 😊

**Comment by elkhalifte** [comment]
Any news @tjoskar :(

**Comment by tjoskar** [comment]
No, not really. This is not easy since the default image is loaded by the browser with the "src" attribute.

What should happen if the default image could not be loaded but the lazyloaded image could be loaded? 

Would this be acceptable:
```
console.log(event.reason);
// output:
// setup
// observer-emit
// start-loading
// mount-image
// loading-succeeded
// default-image-failed

// or:
// setup
// observer-emit
// start-loading
// mount-image
// default-image-failed
// loading-succeeded

// or:
// setup
// observer-emit
// start-loading
// mount-image
// default-image-failed
// loading-failed
```

**Comment by elkhalifte** [comment]
Yeah , i think this would work for me.

**Comment by elkhalifte** [comment]
// setup
// observer-emit
// start-loading
// mount-image
// default-image-failed
// loading-succeeded

**Comment by rafa-as** [comment]
I think that can be achieved with the [onload js native event](https://www.w3schools.com/tags/ev_onload.asp).
